src/ray/stats/metric_defs.h

```diff
@@ -64,6 +64,26 @@ static Gauge ObjectManagerPullRequests("object_manager_num_pull_requests",
                                        "Number of active pull requests for objects.",
                                        "requests");
 
+static Gauge ObjectDirectoryLocationSubscriptions(
+    "object_directory_subscriptions", "Number of object location subscriptions.",
```

Comment:
Can you update the description to explain what each of them indicates? E.g., if this value is high, that means X.